### PR TITLE
WEBUI-940: add functional tests to compare documents

### PR DESCRIPTION
**Pull request by Pranit-Sotre**

### CODE REVIEW

**Review by rakeshkumar1019 — DISMISSED** [review]
looks good 👍 .

**Review by Nisha-QA — DISMISSED** [review]
Looks good

**Review by poonamyadav252 — DISMISSED** [review]

**Review by rajatgoel-gl — DISMISSED** [review]

File: packages/nuxeo-web-ui-ftest/pages/ui/browser.js [hunk 1]
```diff
@@ -371,4 +371,8 @@ export default class Browser extends BasePage {
     publishDialog.waitForVisible();
     return publishDialog;
   }
+
+  get comparePage() {
+    return this.el.element('nuxeo-page div[class="header"]');
```
Comment (Gabez0r) [inline packages/nuxeo-web-ui-ftest/pages/ui/browser.js]: This helper is not specific enough and can return the wrong page. Consider adding the page element tagName to the query to make it more specific. Something like this:
```suggestion
    return this.el.element('nuxeo-diff-page div.header');
```

**Review by Gabez0r — CHANGES_REQUESTED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by Nisha-QA — APPROVED** [review]
Looks good

**Review by rajatgoel-gl — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**